### DISCUSSION TITLE
blogs/230418-karmada-failover

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5190] [Announcements]
# blogs/230418-karmada-failover

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/blogs/230418-karmada-failover.html

**Comment by NickYadance** [comment]
> 当 conditions 中 type 为 Ready 的 condition 状态为 'False' 时， 执行 UpdateClusterControllerTaint 函数添加 effect 为 NoSchedule 和 NoExecute 的污点。

UpdateClusterControllerTaint里只添加了 NoSchedule 污点，此污点并不会触发eviction，NoExecute是在 taint_manager 里添加的

**Reply by windsonsea** [reply]
Thanks, update in PR #5313
